examples/tools/system/package_manager/conanfile.py

```diff
@@ -0,0 +1,50 @@
+from conan import ConanFile
+from conan.tools.system import package_manager
+from conan.tools.gnu import PkgConfig
+
+required_conan_version = ">=2.0"
+
+
+class SysNcursesConan(ConanFile):
+    name = "ncurses"
+    version = "system"
+    description = "A textual user interfaces that work across a wide variety of terminals"
+    topics = ("curses", "terminal", "toolkit")
+    homepage = "https://invisible-mirror.net/archives/ncurses/"
+    license = "MIT"
+    package_type = "shared-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def layout(self):
```

Comment:
Can be removed

Reply:
Addressed on the commit [1588288](https://github.com/conan-io/examples2/pull/177/commits/1588288ccab4c1c09e875191a15ad2669fddf2e9)